components/fieldgroups/experience-event/industry-verticals/experienceevent-live-event-ticket-purchase.json

```diff
@@ -0,0 +1,184 @@
+{
+  "meta:license": [
+    "Copyright 2022 Bounteous. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-live-event-purchase",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Live Event Ticket Purchase",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Purchase history for tickets to a live event, e.g., a concert or sports game.",
+  "definitions": {
+    "experienceevent-live-event-purchase": {
+      "properties": {
+        "xdm:eventPurchaseDetails": {
+          "type": "object",
+          "properties": {
+            "xdm:ticketPurchaseID": {
+              "title": "Ticket Purchase ID",
+              "description": "Unique ID for the ticket purchase order.",
+              "type": "string",
+              "meta:xdmType": "string"
+            },
+            "xdm:eventID": {
```

Comment:
Can we rename it to "liveEventID" ? Same for eventName ?

Reply:
good catch, updated both of those.